SConscript.env

```diff
@@ -9,26 +9,29 @@ import shutil
 
 from build import util, mixxx, depends
 
+print('SConscript.env cwd: ' + os.getcwd())
 Import('build')
 
 # Grab the created environment from the MixxxBuild
 env = build.env
 
-# Check for old build.h file and delete it if it exists
-defs = str(SCons.Script.File('#src/build.h'))
-if os.path.exists(defs):
-    print("Deleting deprecated build file: %s" % defs)
-    os.remove(defs)
+# We are in the variant_dir (win|mac|lin)XX_build, and the 'src' subdirectory
```

Comment:
I'm not sure, I think if you switch between 32-bit and 64-bit builds on the same checkout it could be handy and @Pegasus-RPG has done that in the past. But I don't think the OS name needs to be there.